cmd/github_receive_adapter/main.go

```diff
@@ -56,8 +56,9 @@ func main() {
 
 	log.Printf("Sink is: %q", *sink)
 
-	ra := &github.Adapter{
-		SinkURI: *sink,
+	ra, err := github.New(*sink)
+	if err != nil {
+		log.Fatalf("Failed to create github adaptor: %s", err.Error())
```

Comment:
I think you want "adapter": https://writingexplained.org/adapter-vs-adaptor-difference

("Adapter" is overwhelmingly more popular in American English; British English is more evenly split, but there are no specific rules differentiating the two.)

Reply:
every time...

Reply:
fixed.